### PR TITLE
Add solution

### PR DESCRIPTION
**Pull request by IlyaGrynyshyn**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    lst = [Person(person["name"], person["age"]) for person in people]
```
Comment (viktoria-rybenchuk) [inline app/main.py:11]: use descriptive variable name

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
```
Comment (viktoria-rybenchuk) [inline app/main.py:14]: use variable for better formatting

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,18 @@
...
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        get_person = Person.people[person["name"]]
```
Comment (Dimosphen1) [inline app/main.py:13]: The `get_person` variable name is ambiguous, it would be better to use `person_instance` instead.

**Review by Dimosphen1 — APPROVED** [review]
LGTM, well done! One comment regarding variable naming is left for consideration.

**Pull request closed without merging**